tests/zerotier/test_backend.py

```diff
@@ -0,0 +1,311 @@
+import tarfile
+import unittest
+
+from netjsonconfig import ZeroTier
+from netjsonconfig.exceptions import ValidationError
+
+
+class TestBackend(unittest.TestCase):
+    """
+    Tests for ZeroTier backend
+    """
+
+    maxDiff = None
+    _TEST_CONFIG = {
```

Comment:
Let's only keep one configuration here. In a read world scenario, we won't be generating configuration for two networks simultaneously. 

If you did this because other VPN backends also have similar test, then please point them out.